charts/fpl-case-service/values.yaml

```diff
@@ -2,6 +2,9 @@ tags:
   fpl-ccd-stack: false
   ccd-idam-pr: false
```

Comment:
is ccd-idam-pr needed? is it not idam-pr now?

Reply:
Not need, so I will remove.